catalog/spatial/registry/registry-ebrim-transformer/src/main/java/org/codice/ddf/registry/converter/RegistryPackageConverter.java

```diff
@@ -490,24 +497,28 @@ private static void setMetacardEmailAttribute(List<EmailAddressType> emailAddres
 
     private static void setMetacardPhoneNumberAttribute(List<TelephoneNumberType> phoneNumbers,
             String metacardAttribute, MetacardImpl metacard) {
-        List<String> metacardPhoneNumbers = new ArrayList<>();
 
-        for (TelephoneNumberType digits : phoneNumbers) {
+        List<String> metcardPhoneNumbers = new ArrayList<>();
+        if (CollectionUtils.isNotEmpty(phoneNumbers)) {
+            for (TelephoneNumberType digits : phoneNumbers) {
+                metcardPhoneNumbers.add(getPhoneNumber(digits));
 
-            if (StringUtils.isBlank(digits.getExtension())) {
-                metacardPhoneNumbers.add(String.format("(%s) %s",
-                        digits.getAreaCode(),
-                        digits.getNumber()));
-            } else {
-                metacardPhoneNumbers.add(String.format("(%s) %s extension %s",
-                        digits.getAreaCode(),
-                        digits.getNumber(),
-                        digits.getExtension()));
+                if (CollectionUtils.isNotEmpty(metcardPhoneNumbers)) {
+                    metacard.setAttribute(metacardAttribute, (Serializable) metcardPhoneNumbers);
+                }
             }
         }
-        if (CollectionUtils.isNotEmpty(metacardPhoneNumbers)) {
-            metacard.setAttribute(metacardAttribute, (Serializable) metacardPhoneNumbers);
+    }
+
+    private static StringBuilder buildNonNullString(StringBuilder stringBuilder, String format,
+            String attributeValue, String delimeter) {
+        if (StringUtils.isNotBlank(attributeValue)) {
+            if (stringBuilder.length() > 0) {
+                stringBuilder.append(String.format("%s", delimeter));
```

Comment:
I think you can just leave out the String.format in both places in this method and simply do `stringBuilder.append(...)`. The format call is redundant in this case.